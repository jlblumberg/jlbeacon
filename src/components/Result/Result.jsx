import React from 'react';

const Result = (routeResults) => {
  return (
    <div>
      {console.log(routeResults.data)}
    </div>
  )
}

export default Result;
