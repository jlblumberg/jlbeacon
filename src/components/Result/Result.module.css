.result{
  max-width: 30%;
  margin-top: 5%;
  margin-bottom: 5%;
  margin-left: 4%;
  margin-right: 10%;
}

@media (max-width: 770px) {

  .result {
    max-width: 80%;
    margin: 5%;
  }

}
