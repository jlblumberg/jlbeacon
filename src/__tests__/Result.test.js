import React from 'react';
import Result from '../components/Result/Result';
import { shallow } from 'enzyme';

describe('Result', () => {

  it('renders correctly', () => {
    const wrapper = shallow(<Result/>);
    expect(wrapper).toMatchSnapshot();
  });

});