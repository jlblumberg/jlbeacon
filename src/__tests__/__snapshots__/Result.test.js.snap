// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result renders correctly 1`] = `ShallowWrapper {}`;
